spec/factories/mendeley_papers.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :mendeley_paper do
+    title "Musterstudie"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.